### DISCUSSION TITLE
What are vrepl tables and how do I get rid of them?

### DISCUSSION THREAD

**Discussion by sunneydev** [planetscale/discussion#381] [Bug Reports] [answered]
I truncated some of my tables on my dev branch, merged it with prod and now I have tables like these in my database:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/55714870/215351965-ed267b86-f48a-489f-b4e2-c17c0d96f41b.png">

I can't get rid of them because they disappear in the dev branch.

Can someone explain what are they? I haven't been able to find any information regarding them.

**Comment by fooforge** [comment, marked as answer]
Hey @sunneydev,

Tables with the `_vrepl` suffix appear when you use [deploy requests](https://planetscale.com/docs/concepts/deploy-requests). These are temporary tables and they exist so that we can apply your schema changes without downtime.

We clean them up automatically and they do not count against the storage limits of your selected plan. If you want to learn more about how we perform online schema changes, please [refer to our documentation](https://planetscale.com/docs/learn/how-online-schema-change-tools-work).
